packages/react-conformance/src/defaultTests.tsx

```diff
@@ -366,15 +366,22 @@ export const defaultTests: TestObject = {
         };
         const result = render(<Component {...mergedProps} />, renderOptions);
         const rootEl = getTargetElement(testInfo, result, 'className');
+        const portalEl = staticClassNames.getPortalElement && staticClassNames.getPortalElement(result);
 
         const indexFile = require(indexPath);
         const classNamesFromFile = indexFile[exportName];
 
         const expectedClassNames: { [key: string]: string } = staticClassNames.expectedClassNames ?? classNamesFromFile;
-        const missingClassNames = Object.values(expectedClassNames).filter(
+        let missingClassNames = Object.values(expectedClassNames).filter(
           className => !rootEl.classList.contains(className) && !rootEl.querySelector(`.${className}`),
         );
 
+        if (missingClassNames && portalEl) {
```

Comment:
```suggestion
        if (missingClassNames.length && portalEl) {
```

`missingClassNames` will always be truthy, we want to check `length` to know if there's anything potentially missing.